Editor/CourseValidation/Context/ContextExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Innoactive.CreatorEditor.CourseValidation
+{
+    internal static class ContextExtensions
```

Comment:
Couldn't this be in a `BaseContext`class? It is odd to have an extension class for our own internal stuff.

Reply:
No, as context is mostly stored as interface reference not as generic class.